### DISCUSSION TITLE
Is the Serverless func. concurrency billed monthly ?

### DISCUSSION THREAD

**Discussion by ujjwal-kr** [vercel/vercel#4527] [Help] [answered]
In the documentation, its mentioned that
**The maximum number of concurrent executions for a Serverless Function is 1000.**
in the limits page in documentation, can we know the duration ?

**Comment by coetry** [comment, marked as answer]
We don't bill you by anything other than the number of team seats you have for a Pro plan, or any extra Pro features that you add on to your account (or a custom Enterprise agreement).

Our Serverless Function limits are documented [here](https://vercel.com/docs/v2/platform/limits#general-limits).
